### PR TITLE
Update package.json version for semver

### PR DESCRIPTION
**Pull request by shazron**
npm doesn't like versions without major.minor.patch

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "uncrustify",
-  "version": "0.68",
+  "version": "0.68.1",
```
Comment (mihaipopescu) [inline package.json]: Why not `.0` then?

Comment (shazron) [reply package.json]: Can't publish to npm without the .patch version number. 
I thought the version was 0.68.1 already? (at least what I see in the Releases tab in GH and also the tags for the repo)

Comment (mihaipopescu) [reply package.json]: My mistake then! You are right, there has been a precedent having a patch version number as well.

**Comment by shazron** [comment]
> Please make some changes to the file **release-steps.txt** to help us keep the right version number.

Done. I added an extra commit for cleaning up the package.json

**Pull request merged**